tests/pkcli/admin_test.py

```diff
@@ -35,7 +35,7 @@ def test_delete_user():
     with srunit.quest_start() as qcall:
         _is_empty_table(qcall.auth_db.model("JupyterhubUser"))
```

Comment:
_is_empty_table is the incorrect test. Check for this user in the table(s). The assumption that there is only one user in the db is unnecessary.